tools/topology/development/CMakeLists.txt

```
@@ -61,3 +85,43 @@ foreach(tplg ${TPLGS})
 	add_custom_target(topology_${output} DEPENDS ${output}.tplg)
 	add_dependencies(dev_topologies topology_${output})
 endforeach()
+
+# Duplicate of above to handle topologies in parent directory
+foreach(tplg ${TPLGS_UP})
```

@singalsu why not just:
```suggestion
foreach(tplg ${TPLGS} ${TPLGS_UP})
```
... instead of this big copy/paste(/diverge?)